hack/jenkins/e2e.sh

```diff
@@ -222,5 +222,8 @@ fi
 
 ### Clean up ###
 if [[ "${E2E_DOWN,,}" == "true" ]]; then
+    # Sleep before deleting the cluster to give the controller manager time to
+    # delete any cloudprovider resources still around from the last test.
+    sleep 60
```

Comment:
Please add a note as to how the number 60 was arrived at (presumably based on the ELB retry loop frequency), and that it should not be changed independently of that basis.